README.md

```diff
@@ -1,2 +1,58 @@
 # sbt-cross-project
+
+[![Build Status](https://travis-ci.org/scala-native/sbt-cross-project.png?branch=master)](https://travis-ci.org/scala-native/sbt-cross-project)
+
 Cross-platform compilation support for sbt.
+
+## Installation
+
+In `project/plugins.sbt`:
+
+```scala
+resolvers += Resolver.sonatypeRepo("snapshots")
+
+addSbtPlugin("org.scala-js"     % "sbt-scalajs"       % "0.6.13")
+
+// (1) add sbt-cross
+addSbtPlugin("org.scala-native" % "sbt-cross"         % "0.1-SNAPSHOT")
+
+// (2) add sbt-scala-js-cross
+addSbtPlugin("org.scala-native" % "sbt-scalajs-cross" % "0.1-SNAPSHOT")
+
+// (3) add sbt-native
+addSbtPlugin("org.scala-native" % "sbt-native"        % "0.1-SNAPSHOT")
+```
+
+In `build.sbt`:
+
+```scala
+// (4) shadow sbt-scalajs until Scala.js 1.0.0 is released
+import sbtcross.{crossProject, CrossType}
+
+val sharedSettings = Seq(scalaVersion := "2.11.8") // Scala Native only supports 2.11
+
+lazy val bar =
+  // (5) select supported platforms
+  crossProject(JSPlatform, JVMPlatform, NativePlatform) 
+    .crossType(CrossType.Pure) // [Pure, Full, Dummy], default: CrossType.Full
+    .settings(sharedSettings)
+    .jsSettings(/* ... */)
+    .jvmSettings(/* ... */)
+    // (6) configure Scala-Native settings
+    .nativeSettings(/* ... */)
+
+lazy val barJS     = bar.js
+lazy val barJVM    = bar.jvm
+lazy val barNATIVE = bar.native
```

Comment:
Yes, JS and JVM are not really all-caps. They are acronyms. Native is not an acronym.